src/send_packets.c

```diff
@@ -519,13 +519,28 @@ send_packets(tcpreplay_t *ctx, pcap_t *pcap, int idx)
         now_is_now = false;
     }
 
+    /* SKIP PACKETS
+     * Looping over packets to skip
+     */
+    if(ctx->options->skip_pkts > 0) {
+        for (int i = 0; i < ctx->options->skip_pkts && !ctx->abort && 
+              (pktdata = get_next_packet(ctx, pcap, &pkthdr, idx, prev_packet)) != NULL; i++) {
+        }
+    }
+
+
     /* MAIN LOOP 
      * Keep sending while we have packets or until
      * we've sent enough packets
      */
     while (!ctx->abort &&
             (pktdata = get_next_packet(ctx, pcap, &pkthdr, idx, prev_packet)) != NULL) {
 
+        /* Skip packets */
```

Comment:
This also must be copied to `This change must be also implemented in `send_dual_packets()`